src/predictor/gpu_predictor.cu

```diff
@@ -720,7 +917,8 @@ class GPUPredictor : public xgboost::Predictor {
   }
 
  public:
-  explicit GPUPredictor(Context const* ctx) : Predictor::Predictor{ctx} {}
+  explicit GPUPredictor(Context const* ctx)
+      : Predictor::Predictor{ctx}, column_split_helper_{ctx->gpu_id} {}
```

Comment:
Can we pass the `ctx` instead? It contains a `CUDAContext`, which we can query for stream. It doesn't matter at the moment, but we are moving away from the legacy stream, would be great if the new code can use the context by default.

Reply:
Done.